library/bignum_mod_raw.h

```diff
@@ -144,6 +144,26 @@ int mbedtls_mpi_mod_raw_write( const mbedtls_mpi_uint *A,
 
 /* BEGIN MERGE SLOT 2 */
 
+/** \brief  Perform a subtraction of two MPIs and return the modulus
+ *          of the result.
```

Comment:
It feels like something is missing in this description: modulus relative to what?

How about: "Subtract two MPIs, returning the residue modulo the specified modulus"